### DISCUSSION TITLE
State of the Project

### DISCUSSION THREAD

**Discussion by TheAlexLichter** [tonaljs/tonal#335] [Announcements]
Hey all 👋🏻

First of all - thanks to @danigb and all contributors who made TonalJS possible ☺️

I was wondering what the current state of the project is, given that quite some issues and PRs are open and unanswered?

**Comment by danigb** [comment]
I would say it is "slightly" maintained. As you already notice, I spend very small amount of time with it, but I try to solve all bugs.

I do not plan any new functionality, although I don't discard it. Open to collaborators.

**Reply by TheAlexLichter** [reply]
Thanks for the prompt reply ☺️

But that means you'd potentially accept PRs, e.g. ones implementing slash cords (#192 etc.)?

**Reply by danigb** [reply]
yes
